Adyen/Model/BalanceControl/AbstractOpenAPISchema.cs

```diff
@@ -0,0 +1,78 @@
+/*
+* Adyen Balance Control API
+*
+*
+* The version of the OpenAPI document: 1
+* Contact: developer-experience@adyen.com
+*
+* NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+* https://openapi-generator.tech
+* Do not edit the class manually.
+*/
+
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Adyen.Model.BalanceControl
+{
+    /// <summary>
+    ///  Abstract base class for oneOf, anyOf schemas in the OpenAPI specification
+    /// </summary>
+    public abstract partial class AbstractOpenAPISchema
+    {
+        /// <summary>
+        ///  Custom JSON serializer
+        /// </summary>
+        static public readonly JsonSerializerSettings SerializerSettings = new JsonSerializerSettings
+        {
+            // OpenAPI generated types generally hide default constructors.
+            ConstructorHandling = ConstructorHandling.AllowNonPublicDefaultConstructor,
+            MissingMemberHandling = MissingMemberHandling.Error,
+            ContractResolver = new DefaultContractResolver
+            {
+                NamingStrategy = new CamelCaseNamingStrategy
+                {
+                    OverrideSpecifiedNames = false
+                }
+            }
+        };
+
+        /// <summary>
+        ///  Custom JSON serializer for objects with additional properties
+        /// </summary>
+        static public readonly JsonSerializerSettings AdditionalPropertiesSerializerSettings = new JsonSerializerSettings
```

Comment:
This returns a JsonSerializerSettings object withMissingMemberHandling.Ignore, let's rename it accordingly. We can add in the summary that it's used for objects with additional properties.

Or even better: create a methodthat returns a JsonSerializerSetting. This method can have different parameters where you can also specifiy the value for MissingMemberHandling --> a factory can create the JsonSerializerSettings.

Reply:
same for:   `static public readonly JsonSerializerSettings SerializerSettings`

Reply:
Could you give me a brief example of how these JsonSerializerSettings are used? Which is used out of the two? SerializerSettings vs. AdditionalPropertiesSerializerSettings

Reply:
They only really get used for one-of models that we use. It basically just checks first if the deserializable json is an exact match to a model, and then if it is a match with additional properties (hence the MissingMemberHandling.Ignore). For this endpoint we don't have any one-of models though, so it doesn't do anything :)